xds/internal/balancer/cdsbalancer/cdsbalancer.go

```diff
@@ -270,6 +272,52 @@ func buildProviderFunc(configs map[string]*certprovider.BuildableConfig, instanc
 	return provider, nil
 }
 
+func outlierDetectionToConfig(od *xdsresource.OutlierDetection) *outlierdetection.LBConfig { // Already validated - no need to return error
+	if od == nil {
+		// "If the outlier_detection field is not set in the Cluster message, a
+		// "no-op" outlier_detection config will be generated, with interval set
+		// to the maximum possible value and all other fields unset." - A50
+		return &outlierdetection.LBConfig{
+			Interval: 1<<63 - 1,
+		}
+	}
```

Comment:
Does this mean we will always have a outlier detection balancer, even when the xDS response doesn't specify the config?

Reply:
Yes, IIUC. (another reason I asked about increasing complexity of system on Monday). The no-op balancer takes away the functionality of the Outlier Detection balancer, and simply acts as a forwarder.

Reply:
> The no-op balancer
But is it a no-op balancer? Are you going to treat `Interval:maxint` differently?
Otherwise it's going to do all the counting, just that it will never use the numbers to eject any connections.

Reply:
True. I think I'll treat Interval:maxint the same, almost logically a no-op, but not quite. WDYT?